src/BitStringAddresses/fockaddress.jl

```diff
@@ -644,13 +684,24 @@ Base.show(io::IO, ::MIME"text/plain", i::FermiFSIndex) = show(io, i)
 
 Iterator over occupied modes in address. `N` is the number of fermions. See [`occupied_modes`](@ref).
 """
-struct FermiOccupiedModes{N,S}<:OccupiedModeIterator
+struct FermiOccupiedModes{N,S} <: ModeIterator
     storage::S
 end
 
 Base.length(::FermiOccupiedModes{N}) where {N} = N
 Base.eltype(::FermiOccupiedModes) = FermiFSIndex
 
+"""
+    FermiUnoccupiedModes{N,S<:BitString}
```

Comment:
```suggestion
    FermiUnoccupiedModes{N}
```
No need to specify the storage type here. The docstring is for the more general type.

Reply:
Perhaps `FermiOccupiedModes{N,S<:BitString}` could also be changed to maintain consistency.

Reply:
I don't understand your comment. Do you mean you also want to change (and generalise) the type definition?

My comment was a suggestion to remove the obsolete reference to `Bitstring` in the doc string for the type. If you agree, you still have to commit the change, as my comment is only a suggestion.

Reply:
I mean perhaps the docstring for `FermiOccupiedModes` should be updated from
```julia
"""
    FermiOccupiedModes{N,S<:BitString}

Iterator over occupied modes in address. `N` is the number of fermions. See [`occupied_modes`](@ref).
"""
```
to
```julia
"""
    FermiOccupiedModes{N}

Iterator over occupied modes in address. `N` is the number of fermions. See [`occupied_modes`](@ref).
"""
```
as well.